go/runtime/host/sgx/epid.go

```diff
@@ -135,8 +137,8 @@ func (ep *teeStateEPID) update(
 	// Decode the AVR so we can do further checks.
 	avr, decErr := cmnIAS.UnsafeDecodeAVR(avrBundle.Body)
 	if decErr == nil && avr.TCBEvaluationDataNumber < quotePolicy.MinTCBEvaluationDataNumber {
```

Comment:
This condition needs to be changed as early TCB updates will always have higher or equal `TCBEvaluationDataNumber`. Instead we should probably check that `avr.ISVEnclaveQuoteStatus` is not one of `QuoteOK` or `QuoteSwHardeningNeeded` and only in this case fall back to "without early updating".